<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>visatom_findatomDlg</class>
 <widget class="QDialog" name="visatom_findatomDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>242</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find...</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="showselectedonlycheckBox">
     <property name="text">
      <string>Show found only</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="groupselcomboBox"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Startat</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="fromatomspinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>9999999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Range</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="widthspinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>9999999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="atominfotreeWidget">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="columnCount">
      <number>2</number>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">2</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>groupselcomboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>visatom_findatomDlg</receiver>
   <slot>OnGroupChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>33</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fromatomspinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>visatom_findatomDlg</receiver>
   <slot>OnStartatomChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>33</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>widthspinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>visatom_findatomDlg</receiver>
   <slot>OnStepChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showselectedonlycheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>visatom_findatomDlg</receiver>
   <slot>OnShowOnlyChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>OnFindatomchanged()</signal>
  <slot>OnGroupChanged(int)</slot>
  <slot>OnStartatomChanged(int)</slot>
  <slot>OnStepChanged(int)</slot>
  <slot>OnShowOnlyChanged(int)</slot>
  <slot>OnToatomChanged(int)</slot>
 </slots>
</ui>
